src/Microsoft.PowerShell.Commands.Management/commands/management/CombinePathCommand.cs

```diff
@@ -29,10 +30,11 @@ public class JoinPathCommand : CoreCommandWithCredentialsBase
         /// <summary>
         /// Gets or sets the childPath parameter to the command
         /// </summary>
-        [Parameter(Position = 1, Mandatory = true, ValueFromPipelineByPropertyName = true)]
+        [Parameter(Position = 1, Mandatory = true, ValueFromPipelineByPropertyName = true, ValueFromRemainingArguments = true)]
         [AllowNull]
         [AllowEmptyString]
-        public string ChildPath { get; set; } = String.Empty;
+        [AllowEmptyCollection]
+        public string[] ChildPath { get; set; } = new string[0];
```

Comment:
> (Unless someone's using the cmdlet directly in C#,but that seems unlikely

Yeah, unlikely for this cmdlet, given it would be easier to use Path.Combine() from C#.